src/relay/qnn/op/op_common.h

```diff
@@ -171,6 +171,11 @@ static inline bool QnnBroadcastRel(const Array<Type>& types, int num_inputs, con
   ICHECK_EQ(types.size(), kNumQnnBinaryOpArgTypes);
 
   // Check the scale and zero point types
+  for (size_t i = 0; i < 8; ++i) {
```

Comment:
Why does this one have so many scales and zero points? The others with 4 type checks make sense, but checking 9 types here is a clear outlier.

Reply:
because this is binary operator, it has scales and zps for both lhs an rhs.

Reply:
Yes, it has tensor, scale, zero point for lhs, rhs, and output, added more comments